web/tests/i18n.test.js

```diff
@@ -109,12 +109,16 @@ run_test("tr_tag", ({mock_template}) => {
             realm_name_in_email_notifications_policy:
                 "Include organization name in subject of message notification emails",
             twenty_four_hour_time: "Time format",
+            dense_mode: "Dense mode",
```

Comment:
I'm not sure how mock_template decides that dense_mode properties should be present after moving the section to `General`. I went through the mock_template documentation on Zulip RTD, but that didn't explain things to me for this case.

Reply:
I'm not sure, but probably the right plan is that this test should be reworked to not do so much mocking anyway.